boards/arm/mimx8mm_phyboard_polis/mimx8mm_phyboard_polis_defconfig

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2022 PHYTEC Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_SERIES_IMX8MM_M4=y
+CONFIG_SOC_MIMX8MM6=y
+CONFIG_BOARD_MIMX8MM_PHYBOARD_POLIS=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_CONSOLE=y
+CONFIG_PINCTRL=y
+CONFIG_GPIO=y
+
+# config needed to not generate an error, when using remoteproc
+CONFIG_OPENAMP=y
```

Comment:
If these KConfigs are only needed when using remoteproc, could you add them to the board's `KConfig.defconfig`, and make them depend on a remoteproc related KConfig value? I'd prefer not to require OpenAMP be enabled for this board.